docs/current_docs/features/services.mdx

```diff
@@ -17,18 +17,24 @@ This makes it possible to:
   - Use those services from the calling host (container-to-host networking)
 - Expose host services for use in a Dagger Function (host-to-container networking).
 
+## Service containers
+
 Services instantiated by a Dagger Function run in service containers, which have the following characteristics:
 
 - Each service container has a canonical, content-addressed hostname and an optional set of exposed ports.
 - Service containers are started just-in-time, de-duplicated, and stopped when no longer needed.
 - Service containers are health checked prior to running clients.
 
+### Use cases
```

Comment:
Maybe move this above "service containers"?

Reply:
Done